### PR TITLE
Implement Validation

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-632](url)

### Changelog / Description
Add Validation every input, you can check validation requirement on this file [https://docs.google.com/spreadsheets/d/11_Stik7dVjr1VByIS1raU75xvhTifTxL5HgQw8ybtd4/edit?usp=sharing](url)

### CODE REVIEW

File: src/views/Dashboard/Doctor/Account/Certification.vue [hunk 1]
```diff
@@ -148,6 +149,19 @@ export default {
     certificationDialog: false,
     isUploading: false,
     isEditCertificationDialog: false,
+    rules: {
```
Comment (agustinustheo) [inline src/views/Dashboard/Doctor/Account/Certification.vue:152]: Please move the rules to `computed`, you can refer to this: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

File: src/components/SetKeystorePasswordDialog.vue [hunk 2]
```diff
@@ -106,11 +107,24 @@ export default {
     passwordConfirm: "",
     showPassword: false,
     showPasswordConfirm: false,
-    passwordRule: (val) => !!val || "Password is required",
+    // passwordRule: (val) => !!val || "Password is required",
     passwordConfirmRule: (password) => (val) =>
       (!!password && password == val) || "Passwords must match.",
     recaptchaVerified: false,
     accountName: "",
+    rules: {
```
Comment (agustinustheo) [inline src/components/SetKeystorePasswordDialog.vue:115]: Please move the rules to `computed`, you can refer to this: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

File: src/views/Dashboard/Doctor/Account/Verification.vue [hunk 3]
```diff
@@ -45,6 +47,12 @@ export default {
   name: 'LabAccount',
   mounted() {
   },
+  data: () => ({
```
Comment (agustinustheo) [inline src/views/Dashboard/Doctor/Account/Verification.vue:50]: Please move the rules to `computed`, you can refer to this: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

**Review by agustinustheo — CHANGES_REQUESTED** [review]
In essence, please move all your rules to `computed`, please refer here: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

File: src/views/Dashboard/Doctor/Account/Certification.vue (hunk 1 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Doctor/Account/Certification.vue]: done, all rules move to computed

File: src/components/SetKeystorePasswordDialog.vue (hunk 2 above)
Comment (luthfiazhari) [reply src/components/SetKeystorePasswordDialog.vue]: done, all rules move to computed

File: src/views/Dashboard/Doctor/Account/Verification.vue (hunk 3 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Doctor/Account/Verification.vue]: done, all rules move to computed

**Review by agustinustheo — APPROVED** [review]
LGTM!

**Pull request merged**